bojana.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Let's not forget what we learned in the previous iteration. Don't put unnecessary blank spaces if they are not needed.